### DISCUSSION TITLE
Amazing project but was anyone able to generate music like suno.ai does?

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [suno-ai/bark#502] [General]
I am trying but so far nothing like suno.ai does

**Comment by JonathanFly** [comment]
> I am trying but so far nothing like suno.ai does

I doubt you'll get anything close to the newest Suno Chirp model but there is _some_ music hidden away in Bark. An extremely cherrypicked sample showing some musical range. Typically required overloading the prompt with music cues and try tons of variations. For example:

`♩♩ [music][music][music] ♩ Lyrics line 1 ♩ Lyrics ine 2 ♩ Lyrics Line 3 ♩ [music][music][music] ♩♩ `

https://github.com/suno-ai/bark/assets/163408/e6ae5446-22d2-48ea-a8ec-4ef6501ab698

**Reply by BrutalCoding** [reply]
Thanks for sharing, I did not expect this but it's definitely interesting and fun.

**Comment by FurkanGozukara** [comment]
> > I am trying but so far nothing like suno.ai does
> 
> I doubt you'll get anything close to the newest Suno Chirp model but there is _some_ music hidden away in Bark. An extremely cherrypicked sample showing some musical range. Typically required overloading the prompt with music cues and try tons of variations. For example:
> 
> `♩♩ [music][music][music] ♩ Lyrics line 1 ♩ Lyrics ine 2 ♩ Lyrics Line 3 ♩ [music][music][music] ♩♩ `
> 
>  bark_music_test_1.mp4

i see so not possible to make good music :(

for text to voice coqui i think better
